core/jazz_metrics/index.js

```diff
@@ -28,73 +28,73 @@ const request = require('request');
 const utils = require("./components/utils.js"); //Import the utils module.
 const validateUtils = require("./components/validation.js");
 
-function handler (event, context, cb) {
-  var errorHandler = errorHandlerModule();
-  var config = configObj.getConfig(event, context);
+function handler(event, context, cb) {
```

Comment:
Looks like you are overriding the editorconfig, is this intentional?

Reply:
Not really. But I applied a formatting through VSCode formatter. Would that change anything ?

Reply:
Are you using a VSCode extension? [This](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) extension works with editorconfig. Once this extension is in, open your target file and use the shortcut `Shift + Option + F` for the repo formats to be applied.

Reply:
Applied the repo config with editorConfig plugin